e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/embed-parameters.cy.spec.ts

```diff
@@ -104,14 +104,6 @@ H.describeWithSnowplow(suiteTitle, () => {
         .findByLabelText("Product ID")
         .should("contain", "456");
 
-      H.expectUnstructuredSnowplowEvent(
-        {
-          event: "embed_wizard_option_changed",
-          event_detail: "initialParameters",
```

Comment:
We no longer track changes to embed parameters as we've changed to only track the essentials at the end of each step.